crop_classification/deep_learning.py

```diff
@@ -10,28 +10,21 @@
 from config import (
     SHUFFLE_SEEDS,
     DATASET_TO_SIZES,
-    CLASSIFIER_DROPOUT,
-    NUM_CLASSIFICATION_LAYERS,
-    HIDDEN_VECTOR_SIZE,
-    CLASSIFIER_BASE_LAYERS,
-    ENCODER_VECTOR_SIZES,
-    ENCODER_DROPOUT,
     DL_TIML,
 )
 
 from dl import (
-    Classifier,
     train,
     train_timl_model,
-    TaskEncoder,
     TrainDataLoader,
     concatenate_task_info,
+    load_timl_model,
 )
 
 from typing import Optional
 
 
-def run(data_folder: Path = DATAFOLDER_PATH, zero_shot: bool = False) -> None:
+def evaluate_model(data_folder: Path = DATAFOLDER_PATH, zero_shot: bool = False) -> None:
```

Comment:
Thought it might be a little more clear with this name

Reply:
We should make this change in cropharvest too